src/App.js

```diff
@@ -42,9 +43,10 @@ function App() {
       <BrowserRouter>
         <Navbar />
```

Comment:
Looks good! But the navbar is visible even when I don't have a list :)
![Screenshot from 2022-04-17 11-39-08](https://user-images.githubusercontent.com/11340303/163707213-843816e7-8b7d-49be-8bc3-f9e434762e5c.png)